R/length-params.R

```diff
@@ -42,30 +38,28 @@ get_catch_types <- function(log_threshold = logger::DEBUG) {
       name = dplyr::case_when(
         name == "Thunnini" ~ "Tuna",
         name == "Selachimorpha" ~ "Shark",
+        name == "Reptantia" ~ "Nephropidae",
+        name == "Rajiformes" ~ "Myliobatiformes",
```

Comment:
So I had another look and looks like ISSCAAP uses an old taxonomy in which the order Rajiformes encompasses the current Rajiformes as well as Myliobatiformes. Cause they might also catch skates we cannot just exclude Rajiformes I think. Maybe we need to duplicate a row in the spp data frame by hand so that we get the species for both of these orders? Something like:

```
spp <- ... %>%
    dplyr::add_row(
      interagency_code = "SRX",
      isscaap_taxocode.x = "110XXXXXXX",
      name_scientific = "Myliobatiformes"
    )
```

If we do that we also need to remove the `dplyr::filter(dplyr::row_number() == 1)` at the end of this function and modify `retrieve_lengths()` so that it iterates by name and not by interagency code

Reply:
Here I replaced with Myliobatiformes because Rajiformes didn't occur in Timor and Indonesia according to the FishBase database. It sounded strange to me but effectively Rajiformes (and Rajidae, the skates) are rare in tropical shallow waters: https://www.fishbase.se/summary/OrdersSummary.php?order=Rajiformes. Most of Batoidea in Timor are Myliobatiformes

Reply:
Oh awesome!!! That's perfect! Thanks for checking!